catalog-rest-service/src/main/resources/json/schema/type/tableUsageCount.json

```diff
@@ -0,0 +1,81 @@
+{
+  "$id": "https://open-metadata.org/schema/entity/data/tableUsageCount.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Table Usage Count",
+  "description": "This schema defines type of table usage count",
```

Comment:
here we could say something like: this model is the linking between the usage stage and bulksink steps